docs/changelog/2024.md

```diff
@@ -71,43 +71,13 @@ These possibilities are now leveraged in all schema modules.
 - 🎨 Leverage a more concise way for defining registries: [lnschema-core](https://github.com/laminlabs/lnschema-core/pull/452) [findrefs](https://github.com/laminlabs/findrefs/pull/2) [ourprojects](https://github.com/laminlabs/ourprojects/pull/2) [clinicore](https://github.com/laminlabs/clinicore/pull/14) [cellregistry](https://github.com/laminlabs/cellregistry/pull/2) [@sunnyosun](https://github.com/sunnyosun)
 - 🎨 Squash migrations for faster instance creation: [lnschema-core](https://github.com/laminlabs/lnschema-core/pull/453) [bionty](https://github.com/laminlabs/bionty/pull/162) [wetlab](https://github.com/laminlabs/wetlab/pull/75) [@sunnyosun](https://github.com/sunnyosun)
 
-Bionty changes.
-
-- ✨ Add support for local source files in `sources.yaml` [PR](https://github.com/laminlabs/bionty/pull/156) [@mossjacob](https://github.com/mossjacob)
-
-It is now possible to add local parquet files as source using the `file://` prefix in the `sources.yaml` file.
-For example:
-
-```yaml
-Gene:
-ensembl:
-  arabidopsis thaliana:
-    release-57:
-      url: file:///path/to/local/df_arabidopsis thaliana__ensembl__release-57__Gene.parquet
-```
-
-Note that we discourage modifying the `sources.yaml` file because we are planning to deprecate it soon in favor of only modifying `Source` records.
-
-- ✨ Support `taxa="plants"` in `EnsemblGene` [PR](https://github.com/laminlabs/bionty/pull/153) [@mossjacob](https://github.com/mossjacob)
-
-To allow gene information to be fetched from taxa that are not vertebrates,
-[@mossjacob](https://github.com/mossjacob) introduced a new `taxa` parameter for `bt.Organism`.
-This allows us to better support Gene records from other taxa that we are planning to add to `Bionty` soon.
+🍱 Support `taxa="plants"` in `bionty.Gene` [PR](https://github.com/laminlabs/bionty/pull/153) [@mossjacob](https://github.com/mossjacob) [@Zethson](https://github.com/Zethson)
 
 ```python
-ensembl_gene = EnsemblGene(organism="arabidopsis thaliana", version="release-57", taxa="plants")
-df = ensembl_gene.download_df()
+bt.Gene(organism="arabidopsis thaliana", version="release-57", taxa="plants").source.df()
 ```
 
-Use cases.
-
-- A revamped Nextflow guide that makes use of nf-core/scrnaseq [PR](https://github.com/laminlabs/nextflow-lamin/pull/42) [@Zethson](https://github.com/Zethson)
-
-The new {doc}`/nextflow` guide now shows how to register a Nextflow run with inputs & outputs of a [nf-core/scrnaseq pipeline](https://nf-co.re/scrnaseq/latest) run through a registration script that uses lamindb.
-Such an approach can be automated by deploying the showcased script via:
```

Comment:
That's both a technical detail and hypothetical language ("can be automated" implies "it's not yet automated").

Appealing would be: "A revamped Nextflow guide that makes use of nf-core/scrnaseq and is easy to deploy." But I'm also not entirely sure about the "is easy to deploy" part because I don't fully understand why the new guide is "easier to deploy" than the old guide. To me, the new guide has a much more appealing example, but it's still executing a bunch of glue python as it did before. 🤔

Are you saying it's easier to deploy because the glue python is now easier to deploy because it's cleaner separated (sorry about this word, we still haven't changed the language I fear)?

In that case, I'd say something like "A revamped Nextflow guide that makes use of nf-core/scrnaseq and separates glue logic into a dedicated script for easy automation." And we replace "glue logic" with the appropriate term. Looking for this.

Reply:
> But I'm also not entirely sure about the "is easy to deploy" part because I don't fully understand why the new guide is "easier to deploy" than the old guide. To me, the new guide has a much more appealing example, but it's still executing a bunch of glue python as it did before. 🤔

1. The new guide is just much simpler and the old one was convoluted where the script didn't easily give away what happened.
2. The new script takes nf-core much more into account and is therefore easier to copy and adapt.

Reply:
Got it! Yeah, so this is all great and if there is a concise professional way of saying it we can say it in the changelog. I like what you put here in this comment much more than what you put in the markdown.